falco.yaml

```diff
@@ -76,30 +77,68 @@ pipeline:
       # Replace the find_dependency with find_package macro for newer cmake, otherwise it will fail.
       sed -i 's/find_dependency(Protobuf CONFIG)/find_package(Protobuf CONFIG)/' /usr/lib64/cmake/grpc/gRPCConfig.cmake
 
-  - runs: |
-      mkdir -p "${{targets.destdir}}"/etc/falco
-      install -Dm755 ./falco.yaml "${{targets.destdir}}"/etc/falco/falco.yaml
-      sed -e 's/time_format_iso_8601: false/time_format_iso_8601: true/' < "${{targets.destdir}}"/etc/falco/falco.yaml
+data:
+  - name: builds
+    items:
+      falco-standard: Cloud Native Runtime Security; built with the driver loader
+      falco-no-driver: Cloud Native Runtime Security; built with modern eBPF support and without driver loader
 
-  - working-directory: build
+subpackages:
+  - range: builds
+    name: ${{range.key}}
+    description: ${{range.value}}
+    dependencies:
+      runtime:
+        - falco-rules
     pipeline:
       - runs: |
-          cmake \
-            -DCMAKE_INSTALL_PREFIX=/usr \
-            -DCMAKE_INSTALL_LIBDIR=/usr/lib \
-            -DFALCO_ETC_DIR=/etc/falco \
-            -DCMAKE_BUILD_TYPE=Release \
-            -DUSE_BUNDLED_DEPS=Off \
-            -DMINIMAL_BUILD=On \
-            -DBUILD_DRIVER=On \
-            -DBUILD_FALCO_MODERN_BPF=Off \
-            -DBUILD_BPF=Off \
-          ..
+          mkdir -p "${{targets.subpkgdir}}"/etc/falco
+          cp ./falco.yaml "${{targets.subpkgdir}}"/etc/falco/falco.yaml
+          sed -e 's/time_format_iso_8601: false/time_format_iso_8601: true/' < "${{targets.subpkgdir}}"/etc/falco/falco.yaml
```

Comment:
Rather than duplicate this block with `s/destdir/subpkgdir/` I made the standard falco a subpackage also